### ISSUE TITLE
Request: Webhooks

### ISSUE THREAD

**Issue by t1mmen** [basecamp/bcx-api#170]
Even though [it's not currently on the backlog](https://twitter.com/37signals/status/557440686170316800), I'd like to request Webhooks be added to Basecamp.

Specifically, webhooks for the ```/events.json``` endpoint, both on People and on Projects.

This would make realtime integrations possible, which is especially valuable for services like Hipchat and Slack (oh, and my currently unreleased project, that currently polls ```/people/1/events.json``` every X minutes :-)

Thank you for your time!

**Comment by eileencodes** [comment]
Thank you for the suggestion @t1mmen, we'll let you know when this has been added to the API by commenting and closing this issue. Thanks for using Basecamp :smile:

**Comment by chrispruitt** [comment]
Any updates on when this might be added?

**Comment by kgiszewski** [comment]
Bump, +1 for this.

**Comment by poindextrose** [comment]
+1

**Comment by justmiles** [comment]
+1. Could really use this

**Comment by jschumann** [comment]
+1

**Comment by zscheatham** [comment]
Agreed!  This will be a huge help and will allow Basecamp to fit into our project workflow much better!

**Comment by ckpicker** [comment]
+1

**Comment by ilsken** [comment]
+1, avoiding polling would be awesome

**Comment by lpknight21** [comment]
+1 this would be rad.

**Comment by t1mmen** [comment]
Good news, everyone! Webhooks are incoming for Basecamp 3, according to Jason Fried: https://twitter.com/t1mmen/status/646742990069886976

**Comment by chrispruitt** [comment]
Woohoo!!!! Can't wait!

**Comment by yuanbohan** [comment]
:+1: , Can't wait!!

**Comment by MarkusKeySolutions** [comment]
Awesome! Any more details?

**Comment by olivertappin** [comment]
Now Basecamp 3 is released, are webhooks available now?

**Comment by noctivityinc** [comment]
+1 - any update on webhooks for BC3?
